### PR TITLE
Ids hardcoded check not working good fixed

### PR DESCRIPTION
**Pull request by kevinTveizaga**

### CODE REVIEW

File: apex-checks/src/test/java/org/fundacionjala/enforce/sonarqube/apex/checks/HarcodingIdsCheckTest.java [hunk 1]
```diff
@@ -5,34 +5,35 @@
...
+    private static final String THIRD_VARIABLE_ERROR = "Variable \"secondRecord\" contains hard-coded id value.";
+    private static final String FIRST_EXPRESSION_ERROR_MESSAGE = "Statement contains hard-coded value '554654134167168118'.";
+    private static final String SECOND_EXPRESSION_ERROR_MESSAGE = "Statement contains hard-coded value '554546546465456'.";
     
     private SourceFile sourceFile;
     
     @Test
     public void testHardcodedIdsInVariables() {
-        String filePath = "src/test/resources/checks/HarcodedIdsInVariables.cls";
+        String filePath = "src/test/resources/checks/harcodedIdsInVariables.cls";
```
Comment (JimmyRojas) [inline apex-checks/src/test/java/org/fundacionjala/enforce/sonarqube/apex/checks/HarcodingIdsCheckTest.java]: harcodedIdsInVariables

hardcodedIdsInVariables

File: apex-checks/src/test/resources/checks/HarcodedIdsInVariables.cls [hunk 2]
```diff
@@ -1,9 +0,0 @@
-public class DraftArticle {
```
Comment (JimmyRojas) [inline apex-checks/src/test/resources/checks/HarcodedIdsInVariables.cls:1]: harcodedIdsInVariables.cls

hardcodedIdsInVariables.cls

File: apex-checks/src/test/resources/checks/harcodedIdsInVariables.cls [hunk 3]
```diff
@@ -0,0 +1,18 @@
+public class DraftArticle {
```
Comment (JimmyRojas) [inline apex-checks/src/test/resources/checks/harcodedIdsInVariables.cls]: harcodedIdsInVariables.cls

hardcodedIdsInVariables.cls

**Comment by JimmyRojas** [comment]
see  a few comments, JIC you have

apex-checks/src/test/resources/checks/harcodedIdsInVariables.cls
apex-checks/src/test/resources/checks/HarcodedIdsInVariables.cls

those seem to be duplicated, please merged in a single or renamed one of them

File: apex-checks/src/test/resources/checks/HarcodedIdsInVariables.cls (hunk 2 above)
Comment (kevinTveizaga) [inline apex-checks/src/test/resources/checks/HarcodedIdsInVariables.cls:1]: File deleted

File: apex-checks/src/test/java/org/fundacionjala/enforce/sonarqube/apex/checks/HarcodingIdsCheckTest.java [hunk 4]
```diff
@@ -5,34 +5,35 @@
```
Comment (JimmyRojas) [inline apex-checks/src/test/java/org/fundacionjala/enforce/sonarqube/apex/checks/HarcodingIdsCheckTest.java:5]: HarcodingIdsCheckTest.java
HardcodingIdsCheckTest.java

File: apex-checks/src/test/resources/checks/HarcodedIdsInVariables.cls (hunk 2 above)
Comment (JimmyRojas) [inline apex-checks/src/test/resources/checks/HarcodedIdsInVariables.cls:1]: not fixed :(

**Comment by JimmyRojas** [comment]
see a few comments, JIC you have

apex-checks/src/test/resources/checks/harcodedIdsInVariables.cls
apex-checks/src/test/resources/checks/HarcodedIdsInVariables.cls

those seem to be duplicated, please merged in a single or renamed one of them

(second time)

**Pull request merged**